src/main/java/domain/value/validator/WinningNumbersValidator.java

```diff
@@ -0,0 +1,42 @@
+package domain.value.validator;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import ui.message.ExceptionMessage;
+
+public class WinningNumbersValidator {
+    private static final String CHARACTERS_CONTAIN_NUMBER_COMMA = "^[0-9,\\s]+$";
+    public static final int MINIMUM_WINNING_NUMBER = 1;
+    public static final int MAXIMUM_WINNING_NUMBER = 45;
+    private static final int SIZE_OF_WINNING_NUMBERS = 6;
+
+    public void validateOtherCharacterSymbols(String input) {
+        if (!Pattern.matches(CHARACTERS_CONTAIN_NUMBER_COMMA, input)) {
+            throw new IllegalArgumentException(ExceptionMessage.INPUT_ONLY_NUMBERS_AND_COMMA.getMessage());
+        }
+    }
+
+    public void validateSizeOfWinningNumbers(List<Integer> splitWinningNumbers) {
+        if (splitWinningNumbers.size() != SIZE_OF_WINNING_NUMBERS) {
+            throw new IllegalArgumentException(ExceptionMessage.MUST_INPUT_SIX_WINNING_NUMBERS.getMessage());
+        }
+    }
+
+    public void validateRangeOfWinningNumbers(List<Integer> splitWinningNumbers) {
+        if (splitWinningNumbers.stream()
+                .anyMatch(
+                        winningNumber -> winningNumber < MINIMUM_WINNING_NUMBER || winningNumber > MAXIMUM_WINNING_NUMBER)) {
+            throw new IllegalArgumentException(ExceptionMessage.MUST_INPUT_NUMBERS_IN_VALID_RANGE.getMessage());
+        }
+    }
```

Comment:
if문 안에 스트림 구문이 있으니 이걸 인덴트 오버라고 생각해야 되나 의문이 드네요.

또한 if문 안에 스트림 구문으로 인해 가독성이 조금 떨어지는데 메소드 추출을 이용해서 다음과 같이 변경해도 좋을 것 같아요.

```suggestion
    public void validateRangeOfWinningNumbers(List<Integer> splitWinningNumbers) {
        if (splitWinningNumbers.stream().noneMatch(this::isInRange)) {
            throw new IllegalArgumentException(ExceptionMessage.MUST_INPUT_NUMBERS_IN_VALID_RANGE.getMessage());
        }
    }

    private boolean isInRange(Integer winningNumber) {
        return winningNumber >= MINIMUM_WINNING_NUMBER && winningNumber <= MAXIMUM_WINNING_NUMBER
    }
   
```